--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Fira Code-14:antialias=true";
 static int borderpx = 2;
 
 /*
@@ -65,7 +65,7 @@
 static int bellvolume = 0;
 
 /* default TERM value */
-char *termname = "st-256color";
+char *termname = "st-16color";
 
 /*
  * spaces per tab
@@ -86,31 +86,24 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+        /* Lightbox light palette. */
+        "#213538",  /*  0: black     */
+        "#e82d5a",  /*  1: red       */
+        "#6a9c40",  /*  2: green     */
+        "#a08627",  /*  3: yellow    */
+        "#4987b8",  /*  4: blue      */
+        "#aa6d9c",  /*  5: magenta   */
+        "#1d9980",  /*  6: cyan      */
+        "#ede6d3",  /*  7: white     */
+
+        "#000000",  /*  8: brblack   */
+        "#bd6d44",  /*  9: brred     */
+        "#41595c",  /* 10: brgreen   */
+        "#668082",  /* 11: bryellow  */
+        "#7f9192",  /* 12: brblue    */
+        "#827ab5",  /* 13: violet    */
+        "#94a4a5",  /* 14: brcyan    */
+        "#fdf6e4",  /* 15: brwhite   */
 };
 
 
@@ -118,10 +111,10 @@
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+unsigned int defaultfg = 11;
+unsigned int defaultbg = 15;
+static unsigned int defaultcs = 8;
+static unsigned int defaultrcs = 15;
 
 /*
  * Default shape of cursor
